generators/index.js

```diff
@@ -42,10 +43,10 @@ module.exports = class extends Generator {
         message: "Do want to use existing Sass dependencies?",
         choices: [
           {
-            name: "rh-sass",
+            name: "pfe-sass",
             value: {
-              pkg: "@rhelements/rh-sass",
-              path: "rh-sass/rh-sass"
+              pkg: "@pfelements/pfe-sass",
```

Comment:
I think we need to discuss the package name. Not sure if it should be @pfelements. I think it would either be under @patternfly or @patternfly-elements.

Reply:
Updated this to @patternfly now